webapp/src/components/utils/TextUtils.tsx

```diff
@@ -51,7 +51,7 @@ export function formatChatTextContent(messageContent: string) {
 /*
  * Formats text containing `\n` or `\r` into paragraphs.
  */
-export function formatParagraphTextContent(messageContent: string) {
+export function formatParagraphTextContent(messageContent = '') {
```

Comment:
Wouldn't the following be more explicit about the type:

```suggestion
export function formatParagraphTextContent(messageContent: string = '') {
```

Reply:
type isn't needed here since it can be inferred

`Type string trivially inferred from a string literal, remove type annotation.eslint[@typescript-eslint/no-inferrable-types](https://typescript-eslint.io/rules/no-inferrable-types)`